tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -376,6 +376,22 @@ class CompletionArgSuite extends BaseCompletionSuite {
     topLines = Some(3)
   )
 
+  check(
+    "default-args8",
```

Comment:
Looks like we have a similar issue in Scala 3,  It might be somewhere around https://github.com/scalameta/metals/blob/main/mtags/src/main/scala-3/scala/meta/internal/pc/completions/NamedArgCompletions.scala#L63

but for now you could tag it:

```suggestion
    "default-args8".tag(IgnoreScala3),
```

Reply:
Hmmm... weird output from Scala 3. Not exactly the same issue since it still provides reasonable completions, but it is missing just `snd`. Glanced through the Scala 3 codepath for 5 minutes, but I don't immediately see why.

I put in a `combat` just to at least record the fact that Scala 3 is currently producing some of the arg completions (much better than the previous Scala 2 empty behaviour). Hopefully if someone else fixes this at some point, they'll be able to just remove `compat` and my TODO next to it.

Let me know if you prefer to just tag `IgnoreScala3`

Reply:
Raised an issue to not forget about this https://github.com/scalameta/metals/issues/7188